pom.xml

```
@@ -27,7 +27,12 @@
       <license>
           <name>CeCILL-C</name>
           <comments>French equivalent to LGPL</comments>
-          <url>http://www.cecill.info/licences/Licence_CeCILL-C_V1-en.txt</url>
+          <url>http://cecill.info/licences/Licence_CeCILL-C_V1-en.txt</url>
```

While we're at it, could we make this https? There's no auto-redirect from the site, but explicitly specifying the https protocol works.

```suggestion
          <url>https://cecill.info/licences/Licence_CeCILL-C_V1-en.txt</url>
```

---

:+1:

---

Great!